src/backend/src/form-responses/form-responses.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { FormResponse } from 'src/entities/formResponse.entity';
+import { FormResponsesService } from './form-responses.service';
+import { CreateFormResponseDto, CreateMultiFormResponsesDto } from './dto';
+
+@Controller('form-responses')
+export class FormResponsesController {
+  constructor(private readonly formResponsesService: FormResponsesService) {}
+
+  @Post()
+  async create(
+    @Body('userId') userId: number,
+    @Body('questionId') questionId: number,
```

Comment:
This isn't an immediate problem but we should probably add some kind of todo to make sure we add checks to make sure that the questionid is one of a question that actually exist. And that userId hasn't already answered this question

Reply:
Yep. TODO added. We can delegate that to question controller. As for making sure user hasn't answered, we should perhaps update (?) if user has answered.